content/metastore-integrations/docs/hive-metastore.md

```diff
@@ -151,4 +157,213 @@ aws emr-containers start-job-run --cli-input-json file:///Spark-Python-in-s3-hms
 
 ```
 
+### **3-Connect Hive metastore via thrift service hosted on EKS**
+In this example, our Spark application connects to a standalone Hive metastore service (HMS) running in EKS.
+
+Running the standalone HMS in EKS unifies your analytics applications with other business critical apps in a single platform. It simplifies your solution architecture and infrastructure design. The helm chart solution includes autoscaling feature, so your EKS cluster can automatically expand or shrink when the HMS request volume changes. Also it follows the security best practice to manage JDBC credentials via AWS Secrets Manager. However, the solution requires DevOps and k8s skills, also you need to be able to maintain the workload stability and resiliency in EKS.
```

Comment:
You can soften the tone a bit by saying this...
However, you will need combination of analytics and k8s skill to maintain this solution